admin_guide/managing_networking.adoc

```diff
@@ -1121,21 +1119,21 @@ spec:
       port: 5432
 ----
 
-This will allow routers to access the service, but will allow pods in other users'
-namespaces to access it as well. (In general, this should not be a problem though, since
-those pods could normally just access the service via the public router anyway.)
+This allows routers to access the service, but will also allow pods in other
+users' namespaces to access it as well. This should not be a problem though,
+since those pods could normally access the service by using the public router.
 
-Alternatively, you can create a policy allowing full access from the default namespace, as
-in the *ovs-multitenant* plugin:
+Alternatively, you can create a policy allowing full access from the default namespace, as in the *ovs-multitenant* plugin:
 
-. First, a cluster administrator must add a label to the default namespace so it can be
-matched:
+. Add a label to the default namespace. _(Cluster Administrator role is required to add labels to namesapces.)_
```

Comment:
My original text didn't make this clear either, but this step only needs to happen once, ever, for the entire cluster (whereas the other step has to be done for each namespace you want to allow connections into).